indigo_app/templates/indigo_api/task_detail.html

```diff
@@ -75,7 +87,31 @@ <h5>{{ task.title }}</h5>
           </li>
         {% endif %}
       </ul>
-
+      {% get_comment_form for task as form %}
+        <table>
+          <form action="{% comment_form_target %}" method="post">
+            {% csrf_token %}
+            <textarea
+                    id="id_comment"
+                    name="comment"
+                    class="form-control"
+                    required=""
```

Comment:
`required` doesn't need a value, you can just have `required`.

I suggest pulling these onto the same line as the textarea (wrapping if it's too wide), rather than having many lines of attributes.

Reply:
noted.